canonical-websites/docs.snapcraft.io
# Updated year and CDK text

## QA Instructions
1. Pull the branch or run the demo service URL
2. Open any snapcraft.io page
3. Check the footer contains `Charmed Distribution of Kubernetes` and `2019`
4. #winning
